filter_plugins/opg_custom.py

```diff
@@ -113,6 +113,24 @@ def find_rds_instance(rds_list, db_name):
     return db_info
 
 
+def merge_config_dictionaries(*dicts):
+    """
+    Merges n dictionaries of configuration data
+    :param dicts:
+    :return:
+    """
+    res_dict = {}
+
+    if len(dicts) <= 0:
+        return res_dict
+    elif len(dicts) == 1:
+        return dicts[0]
+    else:
+        for dictionary in dicts:
+            res_dict.update(dictionary)
```

Comment:
same as above, if dicts doesn't contain a list here of valid dictionaries, this will raise an exception